ui/src/app/job-list/job-list.component.spec.ts

```diff
@@ -147,6 +156,7 @@ describe('JobListComponent', () => {
     // Back to page 1, which should have the first 3 jobs.
     de.query(By.css('.mat-paginator-decrement')).nativeElement.click();
     fixture.detectChanges();
+    tick();
```

Comment:
Just a note: I've found that have a `tick()` followed by `fixture.detectChanges()` works best for me (sometimes doesn't work when ordered the other way). Not sure if that was specific to my case or a general thing.

Reply:
I think this is good for cases where it's:

- run some view initialization which triggers some async promises
- tick()
- render (promise values now available)

In my case, the problem was actually some random library that paginator depends on setting a timeout after it gets rendered, annoyingly.